src/components/SegmentedProgressBar/SegmentedProgressBar.jsx

```diff
@@ -8,11 +8,13 @@ import dispatchAnalyticsEvent from '../../helpers/analytics';
  *
  * @param {number} current - The index of the current chapter
  * @param {number} total   - The total number of chapters in the form
+ * @param {string} ariaLabel - An override for the default aria label
  */
 export default function SegmentedProgressBar({
   current,
   total,
   enableAnalytics,
+  ariaLabel,
```

Comment:
I _think_ this is possible, [according to the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#earlier_parameters_are_available_to_later_default_parameters).

```suggestion
  ariaLabel = `Step ${current} of ${total}`,
```